sway-lib-std/tests/test_artifacts/balance_contract/Forc.lock

```diff
@@ -0,0 +1,19 @@
+[[package]]
+name = 'balance_contract'
+dependencies = [
+    'core git+https://github.com/FuelLabs/sway-lib-core?branch=master#082bc8eb0616586ac0000825bfe6e9d47b0c713f',
```

Comment:
There are a few lockfiles here that point to a remote source for `core`. That means their manifest files use github instead of local relative path. Please fix.